packages/authorization-updater/src/authorizationService.ts

```diff
@@ -194,5 +247,183 @@ export const authorizationServiceBuilder = (
       );
       logger.info(`Updated Purpose ${purposeId} state for all clients`);
     },
+    async addClient(client: Client, logger: Logger, correlationId: string) {
+      const token = (await refreshableToken.get()).serialized;
+      const headers = getHeaders(correlationId, token);
+      await authMgmtClient.createClient(
+        {
+          name: client.name,
```

Comment:
`PIN-5017` is in progress, IMO we can add the `client.id` in this PR

Reply:
Done